File: log/dbSendLog.js
const insertLog = (dbcn, log_level,current_time, current_service, source_service, API, error_message,  res) => {
    const db_data = {
      log_level: log_level,
        date: current_time,
        current_service: current_service,
        source_service: source_service,
        type_of_request: API,
        content: error_message
      };
    dbcn.query('INSERT INTO test_log3 SET ?', db_data, (err, result) => {
      if (err) {
        console.error('Error inserting data into DB:', err);
        return res.status(500).send('Error inserting data into DB');
      }
      console.log('Inserted:', result.insertId);
    });
  };

module.exports = {
    insertLog
};

File: go/Features/loggingDB/logger/dbSendLog.go
package logger

import (
	"database/sql"
	"fmt"
	"time"
)


const (
    DB_NAME = "test_db3"
)

func SendLogToDB(db *sql.DB, log_level, currentService, sourceService, api, errorMessage string) error {
	currentTime := time.Now()
	query := "INSERT INTO test_log3 (log_level, date, current_service, source_service, type_of_request, content) VALUES (?, ?, ?, ?, ?, ?)"
	_ , err := db.Exec(query, log_level, currentTime, currentService, sourceService, api, errorMessage)
	if err != nil {
		return fmt.Errorf("error inserting log into database: %v", err)
	}

	fmt.Printf("Inserted log successfully")
	return nil
}
